Sources/OpenAI/OpenAI.swift

```diff
@@ -23,14 +23,22 @@ final public class OpenAI: OpenAIProtocol {
         /// API host. Set this property if you use some kind of proxy or your own server. Default is api.openai.com
         public let host: String
         
+        /// API port. Set this property if you use some kind of proxy or your own server. Default is nil
+        public let port: Int?
+        
+        /// API scheme. Set this property if you use some kind of proxy or your own server. Default is https
+        public let scheme: String?
```

Comment:
Nice addition!
Could you please make separate Pull Request with this change, so we can track the changelog?